Sources/Core/Schema.swift

```diff
@@ -131,7 +131,7 @@ extension Schema {
     }
 }
 
-public struct SchemaObjectRoot: Equatable {
+public struct SchemaObjectRoot: Hashable {
```

Comment:
Is this so we can use `SchemaObjectRoot` in a `Set`?

Reply:
Correct. And `Hashable` extends `Equatable`, so we only need the former.